core/kazoo_apps/src/kapps_maintenance.erl

```diff
@@ -1148,14 +1148,17 @@ init_system() ->
 
 -spec check_release() -> 'ok' | 'error'.
 check_release() ->
-    Checks = [fun kapps_started/0
+    Checks = [fun init_system/0
```

Comment:
you'll need toi to call `kz_datamgr:init_dbs()` prior to `init_system` as in `kapps_controller:initialize_kapps()` or move the call to `kz_datamgr:init_dbs()` to `init_system/0`

Reply:
`kapps_controller:initialize_kapps/0` is run on startup every time so we should be good there.

Reply:
yes, and it calls `init_system`,  why calling it here too ?